stages/org.osbuild.nginx.conf

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
+"""
+Write nginx configuration
+"""
+
+import sys
+import os
+
+import osbuild.api
+
+SCHEMA = """
+"additionalProperties": false,
+"properties": {
+  "path": {
+    "type": "string",
+    "description": "Config file location",
+    "default": "/etc/nginx/nginx.conf"
+  },
+  "config": {
+    "type": "object",
+    "properties": {
+      "listen": {
+        "type": "string",
+        "description": "The address and/or port on which the server will accept requests",
+        "default": "*:80"
+      },
+      "root": {
+        "type": "string",
+        "description": "The root directory for requests",
+        "default": "/usr/share/nginx/html"
+      },
+      "pid": {
+        "type": "string",
+        "description": "File that will store the process ID of the main process",
+        "default": "/run/nginx.pid"
+      },
+      "daemon": {
+        "type": "boolean",
+        "description": "Whether nginx should become a daemon",
+        "default": true
+      }
+    }
+  }
+}
+"""
```

Comment:
Ugh. Do we want to mirror the nginx configuration? I guess you discussed this "offlist"? I don't mind much, but it feels weird to proxy a foreign configuration. Meh..

Reply:
I agree, it doesn't feel right, but I don't see a middle solution between mirroring config keys and writing arbitrary text.  I'm open to other suggestions.